Python/ceval.c

```diff
@@ -2777,16 +2776,14 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             assert(PyList_Check(excs));
             PyObject *orig = POP();
 
-            PyObject *val = do_reraise_star(excs, orig);
+            PyObject *val = _PyExc_PrepReraiseStar(orig, excs);
```

Comment:
This is what Mark suggested to call directly with a CALL_METHOD so that we don't need an opcode for this. I still need to think where that method should be because orig is not necessarily an exception group and we don't want to put this method on BaseException.  So that will be in a future PR.

Reply:
There are several other opcodes like this. If we had a LOAD_BUILTIN opcode it could replace LOAD_ASSERTION_ERROR,  IMPORT_NAME, IMPORT_FROM, PRINT_EXPR etc with a function call. There may be others, but those are obviously not performance critical and essentially just call a function.